src/.editorconfig

```diff
@@ -121,6 +121,7 @@ dotnet_diagnostic.IDE0005.severity=suggestion # Remove unnecessary using directi
 dotnet_diagnostic.S4136.severity=suggestion # Method overloads should be grouped together
 dotnet_diagnostic.S1135.severity=suggestion # Complete TODO comments
 dotnet_diagnostic.S112.severity=suggestion # 'System.Exception' should not be thrown by user code.
+dotnet_diagnostic.S1118.severity=suggestion # Only considered for utility classes
```

Comment:
Let's have a quick team discussion on this. You had to add this because of `public partial class Program { }` for unit testing. I think you could also just add an empty protected or private constructor to that class explicitly. Have we made this severity modification in other projects? I don't feel strongly but just wanted to check.

Reply:
https://sonarsource.github.io/rspec/#/rspec/S1118/csharp

I would prefer not to modify this code rule on a general basis. Yes, please see if there is a different resolution, or if not practical, try suppressing the rule in the one file rather than in the global settings.